app/routes/adapter-files.js

```diff
@@ -12,13 +12,23 @@ const Express = require('express');
 const MetaScript = require('metascript');
 
 const Router = Express.Router();
+const partnerName = require('../../bin/index');
 
 // =============================================================================
 // MAIN ////////////////////////////////////////////////////////////////////////
 // =============================================================================
 
 const cwd = Process.cwd();
-const fileNames = Fs.readdirSync(cwd);
+var dir = cwd;
+
+if (partnerName !== undefined) {
+    var partnerFolder = partnerName.replace(/\s/g, '').replace(/([a-z])([A-Z])/g, '$1-$2').toLowerCase();
+
+    // We need to update this line after we know the exact name of mono repo
+    dir = Path.join(cwd.substring(0, cwd.lastIndexOf('mono-repo') + 'mono-repo'.length), partnerFolder);
```

Comment:
cwd returns the location of the npm command, so `npm run bootstrap` will always return the pack where the `package.json` is, not the location where you ran the npm command from. Looking into it, there's actually no easy to way to get the actual cwd where the npm command is being run from. I think we might have to drop that method completely.